test/active_model_serializers/adapter_for_test.rb

```diff
@@ -2,88 +2,82 @@ module ActiveModelSerializers
   class AdapterForTest < ActiveSupport::TestCase
     UnknownAdapterError = ::ActiveModelSerializers::Adapter::UnknownAdapterError
 
-    def setup
-      @previous_adapter = ActiveModelSerializers.config.adapter
-    end
-
-    def teardown
-      ActiveModelSerializers.config.adapter = @previous_adapter
-    end
```

Comment:
Since every tests modifying the config is now wrap into the block, it should be fine.